### ISSUE TITLE
NTR: Fibroblast growth factor 23 hormone high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8418]
New term request Fibroblast growth factor 23 hormone high in blood (segal_181225225035): FGF23  UMLS: 1968909

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000407
- label: Elevated circulating fibroblast growth factor 23 concentration
- parents: [Abnormal circulating hormone concentration (HP:0003117)](https://hpo.jax.org/app/browse/term/HP:0003117)
- pmid: PMID:29892265
- def: The concentration of fibroblast growth factor 23 in the blood circulation is above the upper limit of normal.
- comment: Fibroblast growth factor-23 (FGF23) is a bone-derived hormone suppressing phosphate reabsorption and vitamin D hormone synthesis in the kidney.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8418
- synonyms:
